cli/azd/pkg/tools/azcli/container_registry.go

```diff
@@ -68,12 +84,56 @@ func (crs *containerRegistryService) GetContainerRegistries(
 	return results, nil
 }
 
+func (crs *containerRegistryService) Login(ctx context.Context, subscriptionId string, loginServer string) error {
+	// First attempt to get ACR credentials from the logged in user
+	dockerCreds, tokenErr := crs.getTokenCredentials(ctx, subscriptionId, loginServer)
+	if tokenErr != nil {
+		// If that fails, attempt to get ACR credentials from the admin user
```

Comment:
What do you think about adding a log statement here for `tokenErr`, so that we know we fell back to admin credentials?

Also, is this an important thing for the user to know about? I could see the SP being used to make changes to the `loginServer` an important thing. Maybe this is important enough to also print out to console?